packages/object-utilities/src/js/__protected.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Creates an protected property in the given object
```

Comment:
Please clearly state that the "protected" property is just non-enumerable and not "really" protected.